packages/plugin-ext/src/hosted/node/plugin-activation-events.ts

```diff
@@ -105,7 +105,7 @@ function updateLanguageContributions(languages: PluginPackageLanguageContributio
 function updateNotebookContributions(notebooks: PluginPackageNotebook[], activationEvents: Set<string>): void {
     for (const notebook of notebooks) {
         if (isObject(notebook) && typeof notebook.type === 'string') {
-            activationEvents.add(`onNotebookSerializer:${notebook.type}`);
+            activationEvents.add(`onNotebook:${notebook.type}`);
```

Comment:
`onNotebook` isn't an implicit plugin activation event. See also [this vscode code](https://github.com/microsoft/vscode/blob/316995c8b685f6e824dca36701b52cc2c5506e71/extensions/extension-editing/src/extensionLinter.ts#L21-L22).